docscan/sandbox/request/check/report/breakdown.go

```diff
@@ -53,21 +53,21 @@ func (b *BreakdownBuilder) WithDetail(name string, value string) *BreakdownBuild
 }
 
 // Build creates a new Breakdown
-func (b *BreakdownBuilder) Build() (Breakdown, error) {
-	breakdown := Breakdown{
+func (b *BreakdownBuilder) Build() (*Breakdown, error) {
+	breakdown := &Breakdown{
 		SubCheck: b.subCheck,
 		Result:   b.result,
 		Details:  b.details,
 	}
 
 	err := validate.NotEmpty(breakdown.SubCheck, "Sub Check cannot be empty")
```

Comment:
Not strictly in scope of this ticket, but I find this NotEmpty validator a bit pointless.

I think a better way of doing this would be:
```
if breakdown.SubCheck == nil {
     return nil, ErrEmptySubCheck
}
```
Where ErrEmptySubCheck is a predefined global variable.

Reply:
I agree, seems much clearer to have it inline, and doesn't involve checking for the same error twice. 
Have removed in 456039f

Reply:
(Have kept error message inline based on reasoning above)